float64 time_scaling_factor
--------------------------
bool success
string error

